rasa_core/validator.py

```diff
@@ -0,0 +1,394 @@
+import logging
+import os
+from jsonschema import validate
+import yaml
+from os import listdir
+from os.path import isfile, join
+import argparse
+from rasa_core import utils
+
+
+logger = logging.getLogger(__name__)
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument(
+    '--domain', '-d',
+    type=str, default='domain.yml',
+    help='Path for the domain file'
+)
+
+parser.add_argument(
+    '--stories', '-s',
+    type=str, default='data/stories.md',
+    help='Path for the stories file or directory'
+)
+
+parser.add_argument(
+    '--intents', '-i',
+    type=str, default='data/intents.md',
+    help='Path for the intents file or directory'
+)
+
+parser.add_argument(
+    '--warnings', '-w',
+    action='store_true',
+    default=False,
+    help='Run script with warings'
+)
+
+parser.add_argument(
+    '--no_validate_intents', action='store_true', default=False,
+    help='Skips validations to intents'
+)
+
+parser.add_argument(
+    '--no_validate_utters', action='store_true', default=False,
+    help='Skips validations to utters'
+)
+
+parser.add_argument(
+    '--no_validate_domain', action='store_true', default=False,
+    help='Skips validations to domain'
+)
+
+
+class Validator:
+    domain = ''
+    intents = []
+    stories = []
```

Comment:
Don't need these empty values here, please set everything in `__init__()`

Reply:
solved [here](https://github.com/lappis-unb/rasa_core/blob/24d89908dbbab1081b9195949fb6ff7da05561e9/rasa_core/validator.py#L59)